### ISSUE TITLE
 UIKit3 (beta4): uk-background-fixed don't work in...

### ISSUE THREAD

**Issue by AlexandreZaytsev** [uikit/uikit#2262]
**in Firefox 50.1.0, Safari 5.1.7, IE 11.0.9.xxx  - no fixed**
in Opera  42.0, Chrome 55.0.2.xxx  - ok (background fixed)

ps
```
<!DOCTYPE html>
<html lang="en-gb" class="uk-background-cover uk-background-center-center uk-background-image@s uk-background-fixed " style="background-image: url(flowers.jpg);">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <title>Icon - UIkit tests</title>
		<link rel="stylesheet" href="FontAwesome/css/font-awesome.min.css">
		<link rel="stylesheet" href="css/uikit.min.css" />
		<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js"></script>
        <script src="js/uikit.min.js"></script>
    </head>
...
```

**Comment by svjansen** [comment]
You shouldn't try to add the classes to the <html> tag.

remove
`class="uk-background-cover uk-background-center-center uk-background-image@s uk-background-fixed " style="background-image: url(flowers.jpg);"` from <html> and add it to <body> instead

**Comment by AlexandreZaytsev** [comment]
no,  no fixed
<body  class="...uk-background-fixed...
no fixed
<body><div class="...uk-background-fixed...
...
did not help ((

**Comment by philoest** [comment]
Thanks for reporting I can confirm this within Firefox, we are working on this. 
Please don't use Safari v. 5.1.7 for Windows any longer the official support ended in 2013, so a lot will not work in there.

**Comment by philoest** [comment]
Has been fixed: f9af435
